pontoon/base/models.py

```diff
@@ -232,6 +307,24 @@ def serialize(self):
             'obsolete': self.obsolete,
         }
 
+    def has_changed(self, locale_code):
```

Comment:
> Would these querysets be evaluated only once instead of two times if you moved the .exists() piece right after .filter()?

They're not evaluated when `filter` is called because it's lazy, so they only get evaluated once when `exists` is called. Putting the call on the same line vs the line below is equivalent because assigning the queryset to a variable doesn't evaluate it.